docs/api-reference/extensions/data-filter-extension.md

```diff
@@ -153,6 +154,61 @@ Format:
 * If `filterSize` is `1`: `[softMin, softMax]`
 * If `filterSize` is `2` to `4`: `[[softMin0, softMax0], [softMin1, softMax1], ...]` for each filtered property, respectively.
 
+##### `getFilterCategory` ([Function](../../developer-guide/using-layers.md#accessors), optional) {#getfiltercategory}
```

Comment:
Nit: This name didn't click immediately for me. I would probably have gone with `getDataFilterCategory` (especially as we now have multiple "filter" extensions) but I guess the naming convention is somewhat fixed by now.

Reply:
I don't love it either, I was trying to stay "consistent" with `getFilterValue`. I think we should stick with this or make both accessors consistent, e.g. use `getDataFilterValue` & `getDataFilterCategory`. This would be a breaking change, and I'm not sure it is worth it